presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java

```diff
@@ -190,6 +190,46 @@ public void testFiltersAndProjections()
         assertQuery("SELECT if(orderkey % 2 = 0, quantity_by_linenumber) FROM orders_ex");
     }
 
+    @Test
+    public void testAnalyzeStats()
+    {
+        assertUpdate("ANALYZE region", 5);
+        // presto:tpch> show stats for region;
```

Comment:
This comment can be removed I think. Only please add a comment that the query results in a table giving the (column_name, data_size, distinct_values_count, nulls_fraction, row_count, low_value, high_value).